.eslintrc.js

```diff
@@ -1,124 +1 @@
-var path = require('path');
-var OFF = 0;
-var WARNING = 1;
-var ERROR = 2;
-
-module.exports = {
-  env: {
-    es6: true,
-    browser: true,
-    node: true,
-  },
-  overrides: [
-    {
-      files: ['*.spec.js'],
-      env: {
-        es6: true,
-        browser: true,
-        node: true,
-        jest: true,
-      },
-      parserOptions: {
-        ecmaVersion: 8,
-      },
-    },
-  ],
-  parserOptions: {
-    sourceType: 'module',
-    ecmaVersion: 2018,
-    ecmaFeatures: {
-      impliedStrict: true,
-    },
-  },
-  globals: {
-    __version: true,
-    __coreAPISpec: true,
-    __filename: true,
-    __publicPath: true,
-    __copyrightYear: true,
-  },
-  extends: [
-    'eslint:recommended',
-    'plugin:vue/recommended',
-    'plugin:import/errors',
-    'plugin:import/warnings',
-    'prettier',
-  ],
-  plugins: ['import', 'vue', 'kolibri'],
-  settings: {
-    'import/resolver': {
-      [path.resolve(
-        path.join(path.dirname(__filename), './frontend_build/src/alias_import_resolver.js')
-      )]: {
-        extensions: ['.js', '.vue'],
-      },
-    },
-  },
-  rules: {
-    'comma-style': ERROR,
-    'max-len': [
-      ERROR,
-      100,
-      {
-        ignoreStrings: true,
-        ignoreTemplateLiterals: true,
-        ignoreUrls: true,
-      },
-    ],
-    'vue/attribute-hyphenation': [ERROR, 'never'],
-    'vue/require-default-prop': OFF,
-    'vue/html-self-closing': [
-      ERROR,
-      {
-        html: {
-          void: 'never',
-          normal: 'never',
-          component: 'always',
-        },
-        svg: 'always',
-        math: 'always',
-      },
-    ],
-    'vue/max-attributes-per-line': [
-      ERROR,
-      {
-        singleline: 5,
-        multiline: {
-          max: 1,
-          allowFirstLine: false,
-        },
-      },
-    ],
-    'vue/html-closing-bracket-newline': [
-      ERROR,
-      {
-        singleline: 'never',
-        multiline: 'always',
-      },
-    ],
-    'vue/component-name-in-template-casing': [
-      ERROR,
-      'PascalCase',
-      {
-        ignores: [
-          'mat-svg',
-          'file-svg',
-          'component',
-          'transition',
-          'transition-group',
-          'router-link',
-          'router-view',
-        ],
-      },
-    ],
-    'vue/no-spaces-around-equal-signs-in-attribute': ERROR,
-    'import/first': ERROR,
-    'import/no-duplicates': ERROR,
-    'import/newline-after-import': ERROR,
-    'import/order': ERROR,
-
-    // Custom vue rules
-    'kolibri/vue-filename-and-component-name-match': ERROR,
-    'kolibri/vue-component-registration-casing': ERROR,
-  },
-};
+module.exports = require('kolibri-tools/.eslintrc');
```

Comment:
As far as I can tell, vue files are no longer being linted after this change. This is true both of `<script>` and `<template>` sections.

To test this, try adding `props: ['simpleprop']` to a component before and after this PR. Before, it will complain

```
eslint:vue/require-prop-types Props should at least define their types.
```

but after it will not.

Currently I believe this is a config issue and not related to `lint.js` because the behavior happens whether I run the `devserver` or try to set up linting in sublimetext.

Investigating further...

Reply:
Confirmed that moving `.eslintrc.js` back to the root directory and updating the `import/resolver` allows `eslint` to work correctly in Sublime.

However, even with this change, `yarn run lint` behaves strangely:

* `<template>` errors are thrown without context, e.g. `Kolibri linter: Invalid array length` (which I believe comes from `HTMLHint`, not `eslint`)
* Vue style rules like `require-prop-types` are not reported
* Syntax errors in the `<script>` section are reported, but again lack context (filename)

Reply:
I hate magic import resolvers. Will play around with this and try to add some testing.

Will look at the context issues again, I was trying to use the built in reporters for each linter to maintain context as much as possible, but it's possible that invoking them with the file contents and not the file itself may be causing issues. Thanks for checking this out!

Reply:
I have tested this out, and this all seems to be working properly now, and eslint is working correctly in Sublime for me also. Let me know if this is still causing issues as the PR is now.

Reply:
working!